apps/web/modules/videos/views/videos-single-view.tsx

```diff
@@ -42,88 +43,149 @@ export default function JoinCall(props: PageProps) {
+              accent: "#FFF",
+              accentText: "#111111",
+              background: "#111111",
+              backgroundAccent: "#111111",
+              baseText: "#FFF",
+              border: "#292929",
+              mainAreaBg: "#111111",
+              mainAreaBgAccent: "#1A1A1A",
+              mainAreaText: "#FFF",
+              supportiveText: "#FFF",
+            },
           },
-        },
-        showLeaveButton: true,
-        iframeStyle: {
-          position: "fixed",
-          width: "100%",
-          height: "100%",
-        },
-        url: meetingUrl,
-        userName: overrideName ?? loggedInUserName ?? undefined,
-        ...(typeof meetingPassword === "string" && { token: meetingPassword }),
-        ...(hasTeamPlan && {
-          customTrayButtons: {
-            ...(showRecordingButton
-              ? {
-                  recording: {
-                    label: "Record",
-                    tooltip: "Start or stop recording",
-                    iconPath: RECORDING_DEFAULT_ICON,
-                    iconPathDarkMode: RECORDING_DEFAULT_ICON,
-                  },
-                }
-              : {}),
-            ...(showTranscriptionButton
-              ? {
-                  transcription: {
-                    label: "Cal.ai",
-                    tooltip: "Transcription powered by AI",
-                    iconPath: TRANSCRIPTION_STOPPED_ICON,
-                    iconPathDarkMode: TRANSCRIPTION_STOPPED_ICON,
-                  },
-                }
-              : {}),
+          showLeaveButton: true,
+          showUserNameChangeUI: shouldShowUserNameChangeUI,
+          iframeStyle: {
+            position: "fixed",
+            width: "100%",
+            height: "100%",
           },
-        }),
-      });
+          url: meetingUrl,
+          userName: userName,
+          ...(typeof meetingPassword === "string" && { token: meetingPassword }),
+          ...(hasTeamPlan && {
+            customTrayButtons: {
+              ...(showRecordingButton
+                ? {
+                    recording: {
+                      label: "Record",
+                      tooltip: "Start or stop recording",
+                      iconPath: RECORDING_DEFAULT_ICON,
+                      iconPathDarkMode: RECORDING_DEFAULT_ICON,
+                    },
+                  }
+                : {}),
+              ...(showTranscriptionButton
+                ? {
+                    transcription: {
+                      label: "Cal.ai",
+                      tooltip: "Transcription powered by AI",
+                      iconPath: TRANSCRIPTION_STOPPED_ICON,
+                      iconPathDarkMode: TRANSCRIPTION_STOPPED_ICON,
+                    },
+                  }
+                : {}),
+            },
+          }),
+        });
+
+        if (userName) {
+          callFrame.setUserName(userName);
+        }
 
-      if (overrideName) {
-        callFrame.setUserName(overrideName);
```

Comment:
Another thing is if overrideName name is passed we don't want to display the login modal and ask for name anywhere